CHANGELOG.md

```diff
@@ -1,5 +1,337 @@
+
+  ```toml
+  # wrangler.toml
+
+  [[kv-namespaces]]
+  binding = "TODOS"
+  id = "0f2ac74b498b48028cb68387c421e279"
+  ```
+
+  **If you were previously using the undocumented `kv-namespaces` support in your project config, you'll need to make a few changes to your project to use Wrangler 1.1.0!** KV namespaces now need to be created manually, in the Cloudflare Dashboard, to be able to use them in your Wrangler configuration - previous versions of Wrangler created the namespace for you, but the process is now manual, to allow developers to be more explicit about how their KV namespaces are created.
+
+  [ashleymichal]: https://github.com/ashleymichal
+  [pull/334]: https://github.com/cloudflare/wrangler/pull/334
+
+  - **add check + error message for pre 1.1.0 kv namespace format - [xortive], [pull/369]**
+
+  For users of the previously undocumented `kv-namespaces` functionality in Wrangler, we've provided a warning and upgrade path, to help you upgrade your KV configuration in your project to the correct format.
+
+  [xortive]: https://github.com/xortive
+  [pull/369]: https://github.com/cloudflare/wrangler/pull/369
+
+  - **feat: unify upload form building - [ashleymichal], [pull/329]**
+
+  This PR brings consistency to the way that metadata is handled during Wrangler's `preview` and `build` commands: previously, a `metadata.json` file handled the bindings for your Wrangler project, but because it was handled differently for various project types (such as `webpack` or `rust`), it led to inconsistent behavior during the upload process. This PR removes usage of `metadata.json` in favor of building metadata for each project type in-memory, which will then be uploaded directly to Workers platform. This work is foundational for improved Workers KV support in Wrangler 🙌
+
+  [ashleymichal]: https://github.com/ashleymichal
+  [pull/329]: https://github.com/cloudflare/wrangler/pull/329
+
+  - **Use stdin instead of arguments for wrangler config - [xtuc], [pull/329]**
+
+  We've made the `wrangler config` command interactive – the previous version of the command, `wrangler config $email $apiKey`, would be captured by your terminal's history, often exposing that information in a `~/.bash_history` or a similar file. The new version of `wrangler config` will prompt you for your `email` and `api_key` via `stdin`.
+
+  In addition, this PR also adds support for a `WRANGLER_HOME` environment variable, which will be the location for Wrangler's "home" directory, if you need to customize where Wrangler saves its configuration information.
+
+  [xtuc]: https://github.com/xtuc
+  [pull/329]: https://github.com/cloudflare/wrangler/pull/239
+
+  - **Settings: make global config file optional - [AaronO], [pull/225]**
```

Comment:
i'd bump this up in the list since it's pretty cool

Reply:
i moved it to the second spot on features, sound good to you?

Reply:
sorry, meant to add this in the review i just did

Reply:
i would change this title to "use ENV variable to configure wrangler"